docs/extensions/intermediate/controllers-routes.md

```diff
@@ -206,3 +206,5 @@ class DropBearController implements ControllerProviderInterface
 }
 ```
 
+The callback method can then be accessed via `/dropbear/koala/{type}` (frontend) and `/bolt/dropbear/koala/{type}` (backend).
```

Comment:
IMO it should be "`/dropbear/koala/{type}` (for frontend controllers) or `/bolt/dropbear/koala/{type}` (for backend controllers)".